edu.harvard.i2b2.ontology/gensrc/edu/harvard/i2b2/ontology/datavo/vdo/GetChildrenType.java

```diff
@@ -28,6 +28,7 @@
  *         &lt;element name="parent" type="{http://www.w3.org/2001/XMLSchema}string"/>
  *       &lt;/sequence>
  *       &lt;attGroup ref="{http://www.i2b2.org/xsd/cell/ont/1.1/}return_attributeGroup"/>
+ *       &lt;attribute name="numLevel" type="{http://www.w3.org/2001/XMLSchema}int" default="1" />
```

Comment:
We think the default should be -1 so it's not executed if numLevel isn't present?

Reply:
The default should be **1** for one level below the parent.  Otherwise, the current i2b2 will not work.